src/mocks/idlemembers.json

```diff
@@ -0,0 +1,5 @@
+{
+  "idleMembers": {
+    "idleMemberUserNames": ["rohan-rajgupta", "sumit", "rohit", "tanya", "akshay", "shubham", "devashish", "lakshay", "rucha", "swebert", "pavan", "nikhil", "ankur", "swaraj"]
+  }
+}
```

Comment:
We have added API response mocking in the codebase, please refer to [this](https://mswjs.io/docs/api/rest) doc to learn to write the handler for the same.

Check [this message on discord](https://discord.com/channels/673083527624916993/741616265050849282/937589378593726495).
Please go through [CONTRIBUTING.md](https://github.com/Real-Dev-Squad/website-status/blob/develop/CONTRIBUTING.md#to-run-the-project-with-mocked-apis)